exo.jcr.docs/exo.jcr.docs.developer/en/src/main/docbook/en-US/modules/jcr/configuration/workspace-persistence-storage.xml

```diff
@@ -134,6 +134,15 @@
     url="http://en.wikipedia.org/wiki/Bloom_filter&quot; &gt;http://en.wikipedia.org/wiki/Bloom_filter">here</ulink>.</para>
   </note>
 
+  <note>
+   <para>
+     It's possible to force the clean swap files thanks to the System property exo.jcr.spoolConfig.force.clean.swap.
+     The default value of this parameter is false (force clean disabled).
+     If force clean is enabled, It's possible to define the live time (in seconds) of swap files thanks to the System property exo.jcr.spoolConfig.swap.live.time.
+     The default value of this parameter is -1 (disabled).
```

Comment:
We should add :
* what happens if we do not enabled this property
* what is the role of these swap files
* what are the benefits/drawbacks of setting a low/high value for exo.jcr.spoolConfig.swap.live.time

If will help the customer to decide which values to set.